packages/taro-extend/src/index.d.ts

```
@@ -0,0 +1,1665 @@
+     * Create and initialize a DOM event of the specified type. If a properties object is given, use it to extend the new event object. The event is configured to bubble by default; this can be turned off by setting the bubbles property to false.
+     * An event initialized with this function can be triggered with trigger.
+     * @param type
+     * @param properties
+     * @return
+     */
+    Event(type: string, properties: any): Event;
+
+    /**
+     * Get a function that ensures that the value of this in the original function refers to the context object. In the second form, the original function is read from the specific property of the context object.
+     */
+    proxy(fn: Function, context: any): Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)